### PR TITLE
Read functionality

### PR DESCRIPTION
**Pull request by ToddComte**
We are awesome

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -17,10 +17,30 @@ import {
...
+    this.readCollections()
+  }
+  
   render() {
+    console.log(this.state.collections)
+    const { logged_in, new_user_route, sign_in_route, sign_out_route, current_user } = this.props
     return (
       <Router>
-        <Header />
+        <Header logged_in={logged_in} new_user_route={new_user_route} sign_in_route={sign_in_route} sign_out_route={sign_out_route} current_user={current_user}/>
```
Comment (elysemontano) [inline app/javascript/components/App.js:43]: This could be separated into multiple lines for readability.

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -17,10 +17,30 @@ import {
...
 class App extends Component {
+  constructor(props){
+    super(props)
+      this.state = {
+        collections: [] 
+      }
+    
+  }
+
+  readCollections = async () =>{
```
Comment (elysemontano) [inline app/javascript/components/App.js:28]: Is everything rendering on the page properly?

**Review by elysemontano — APPROVED** [review]
Left a few comments to check, otherwise looks good

File: app/javascript/components/App.js (hunk 1 above)
Comment (celoazevedo) [reply app/javascript/components/App.js:43]: Will do!

File: app/javascript/components/App.js (hunk 2 above)
Comment (celoazevedo) [reply app/javascript/components/App.js:28]: We were able to log the changes in state collections property. We have not worked on the display yet.

**Pull request merged**